### ISSUE TITLE
Integrate everything search in the open shell search bar

### ISSUE THREAD

**Issue by ibay613** [Open-Shell/Open-Shell-Menu#836]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

Make an additional option to use the everything search engine in openshell instead of the default windows one.

**Describe alternatives you've considered**

**Additional context**

**Comment by Gaurav-Original-ClassicShellTester** [comment]
Have you read this? :) http://www.classicshell.net/forum/viewtopic.php?f=4&t=1588#SearchProviders

**Comment by ibay770** [comment]
Thank you, yes, thats not perfect, but until we get get integration fixed, I'll have to stick with this,

**Comment by aetonsi** [comment]
Yes i concur, the Everything Toolbar works, but an integration into Open-Shell would be better..
Also, i'm not able to understand how to add a "search provider" for _Open_-Shell, as every tutorial is for _Classic_ Shell, and this here are a bit different..

**Comment by aetonsi** [comment]
@Gaurav-Original-ClassicShellTester have you succeeded in adding a search provider? could you kindly explain the steps?

**Comment by refriedfood** [comment]
How do I configure this? It would be amazing!

**Comment by Venthe** [comment]
Sorry to necro' but the instruction in http://www.classicshell.net/forum/viewtopic.php?f=4&t=1588#SearchProviders works well. 

![image](https://github.com/user-attachments/assets/d68a7500-8269-4374-b475-7db41de1328f)

![image](https://github.com/user-attachments/assets/272d3c43-e1ab-4245-80ee-f03da7d9cd05)

![image](https://github.com/user-attachments/assets/3fe69c5c-b504-41ac-afcc-bd3a9503a476)

It's not _the_ integration I'd wish for, but this is the integration Classic/Openshell gives

**Comment by PCMasterRays** [comment]
Thanks for this, judging by this still being open, there's hope? I wanna say, some of us are really wanting official "everything" support, this would make Windows 11 level up (alongside other tweaks) a LOT <3

Thanks for this one for now @Venthe
